ProjectFiles/Assets/Scripts/Game/RoundManager.cs

```diff
@@ -117,15 +121,16 @@ private void EndRoundEvent()
         {
             NotifyRoundEnd();
             roundNumber++;
-            if (roundNumber < maxRounds)
+            if (roundNumber < RoundProperties.MaxRounds && world.GetNumPlayers() > 1)
```

Comment:
This logic isn't going to work. The world.GetNumPlayers won't go below 1 until the players have been eliminated, so whatever happens there will have to be one more round with just 1 player before the game ends.